package.json

```diff
@@ -39,15 +39,15 @@
     "src"
   ],
   "dependencies": {
-    "dragselect": "^2.7.4"
+    "dragselect": "^3.0.4"
```

Comment:
I don't see how the update to dragselect was required to fix the Blockly related issue.

Reply:
This is not related to the fix but rather purely a full version bump for the release process (to avoid possible depreciation and vulnerabilities)